src/controllers/trip.controller.js

```diff
@@ -39,27 +39,42 @@ class tripController {
       const status = 'pending';
       const { type } = req.body;
 
-      const createdTrip = await tripService.CreateTrip(req, req.body, tripid, type);
+      const createdTrip = await tripService.CreateTrip(
+        req,
+        req.body,
+        tripid,
+        type
+      );
```

Comment:
Unnecessary change. I think we might have the previous code we don't need to change it ?